### PR TITLE
enhanced pincode.txt

### PR DESCRIPTION
**Pull request by saurabhjn76**
Fixes issue #218 

Etherpad Link: [http://dream.susi.ai/p/longlat](http://dream.susi.ai/p/longlat)

Changes: added intent in pincode.txt in knowledge 

Screenshots for the change: 
image

### CODE REVIEW

**Comment by Manveer695** [comment]
I have no idea how to solve it but I am getting old answers:
longlat issue

Can you check it again?

**Comment by saurabhjn76** [comment]
@Manveer695 it's running. The difference was because of `pin code ` instead of ` pincode`, I will add` pin code` in skill too. Thanks :+1: 
image

**Comment by Manveer695** [comment]
@saurabhjn76 oh great!

**Review by dynamitechetan — APPROVED** [review]

**Review by Laughing-Boy — APPROVED** [review]

**Comment by mariobehling** [comment]
Please add examples, description and images.

**Comment by mariobehling** [comment]
Ensure format is followed and image is added.

**Comment by mariobehling** [comment]
Provide a proper name. Don't user Yahoo Query Language something. You don't name a skill that uses JSON "JSON Skill Something".

**Comment by saurabhjn76** [comment]
ok :+1:

File: models/general/knowledge/en/pincode.txt [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
 {
 "url":"http://api.geonames.org/postalCodeLookupJSON?postalcode=$1$&username=demo",
 "path":"$.postalcodes[0]"
 }
+eol
+
+# Returns the long lat values based on zipcode and country
+tell the longitude and latitude in * with pincode *|do you know the longitude and latitude in * with pincode *|find the longitude and latitude in * with pincode *|Search the longitude and latitude in * with pincode *|What is the longitude and latitude in * with pincode *|tell the longitude and latitude in * having pincode *|do you know the longitude and latitude in * having pincode *|find the longitude and latitude in * having pincode *|Search the longitude and latitude in * having pincode *|What is the longitude and latitude in * having pincode *
+!example:tell the longitude and latitude in India having pincode 160019 
+!console:Latitude:$latitude$ and Longitude:$longitude$
```
Comment (AliAyub007) [inline models/general/knowledge/en/pincode.txt]: Also add !expect: answer expected from your example

Comment (saurabhjn76) [reply models/general/knowledge/en/pincode.txt]: @AliAyub007 done :+1:  Thanks

**Review by AliAyub007 — APPROVED** [review]

**Comment by hpdang** [comment]
Thank you!

**Pull request merged**